### ISSUE TITLE
Edge in conflict with JS backticks

### ISSUE THREAD

**Issue by thetutlage** [edge-js/edge#56]
_From @clayderson on September 19, 2018 2:9_

If I try to use the backticks, an error message appears when I try to access the page

Example

```
player.attr('src', `https://example.com/renders/body/${data.uuid}?scale=4`)
```

![captura de tela 2018-09-18 23 08 39](https://user-images.githubusercontent.com/15388584/45726903-cb4bd300-bb97-11e8-869d-7a1ede9cbb2e.png)

_Copied from original issue: adonisjs/adonis-framework#951_

**Comment by ramireznoy** [comment]
Yes, it is veery annoying not to have template strings available on edge templates code :-( Is there any option or setting to make them work?

**Comment by thetutlage** [comment]
Fixed in `v3.0.0`. Docs for the release will be out by the end of the week
